xbmc/LangInfo.h

```diff
@@ -76,9 +99,28 @@ class CLangInfo : public ISettingCallback
   const std::string& GetTimeZone() const;
 
   const std::string& GetRegionLocale() const;
-  const std::string GetLanguageLocale(bool twochar = false) const;
 
-  bool ForceUnicodeFont() const { return m_currentRegion->m_forceUnicodeFont; }
+  /*!
+  \brief Returns the full locale of the current language.
+  */
+  const std::string& GetLocale() const;
+
+  /*!
+   \brief Returns the two character ISO 639-1 language code of the current language.
+   */
+  const std::string& GetLanguageLocale() const;
+
+  /*!
+   \brief Returns the <ll>_<TT> locale of the current language.
+   */
+  const std::string& GetLanguageTerritoryLocale() const;
```

Comment:
These names are very confusing. What's the difference between `GetLocale` and `GetLanguageTerritoryLocale`? And I realize this old names but "LanguageLocale" and "RegionLocale" makes very little sense as locale is by definition both of those (GetLanguageLocale as you used below is slightly better)